CMakeLists.txt

```diff
@@ -676,6 +679,23 @@ if (CAPSTONE_LOONGARCH_SUPPORT)
             )
 endif ()
 
+if(CAPSTONE_XTENSA_SUPPORT)
+    add_definitions(-DCAPSTONE_HAS_XTENSA)
+    set(SOURCES_XTENSA
+        arch/Xtensa/XtensaDisassembler.c
+        arch/Xtensa/XtensaInstPrinter.c
+        arch/Xtensa/XtensaMapping.c
+        arch/Xtensa/XtensaModule.c
+    )
+    set(HEADERS_XTENSA
+        arch/Xtensa/XtensaDisassembler.h
+        arch/Xtensa/XtensaInstPrinter.h
+        arch/Xtensa/XtensaMapping.h
+        arch/Xtensa/XtensaModule.h
+    )
+    set(TEST_SOURCES ${TEST_SOURCES} test_xtensa.c)
```

Comment:
Delete the `test_xtensa` files (Python and C) and move the instruction tests to `tests/details/xtensa.yaml`